src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Test/ScenarioTests/AzureFiles/ItemTests.ps1

```diff
@@ -229,4 +236,90 @@ function Test-AzureFSGetRPs
 	{
 		Cleanup-Vault $vault $item $container
 	}
+}
+
+function Test-AzureFSFullRestore
+{
+	try
+	{
+		$vault = Get-AzureRmRecoveryServicesVault -ResourceGroupName $resourceGroupName -Name $vaultName
+		Get-AzureRmRecoveryServicesVault -ResourceGroupName $resourceGroupName -Name $vaultName | Set-AzureRmRecoveryServicesVaultContext
```

Comment:
We should not be using Set-AzureRmRecoveryServicesVaultContext anywhere any more. This is deprecated. We need to use -VaultId parameter instead. Please make this change in all the AFS tests.

Reply:
Without this, it fails here(https://github.com/MabOneSdk/azure-powershell/blob/62b7c47d932731c467086bb878855ec274a2b27c/src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.ServiceClientAdapter/BMSAPIs/RestoreDiskAPIs.cs#L43)

Reply:
Then it's a bug. Can you please track why this is happening and fix it? We shouldn't use Set-Context cmdlet any more.